专栏的PDF每页前几行被banner遮挡

![image](https://user-images.githubusercontent.com/6437705/82724317-c1cd2380-9d07-11ea-9adb-426dab9582bd.png)
![image](https://user-images.githubusercontent.com/6437705/82724326-d4475d00-9d07-11ea-8718-f7e20bca4207.png)


![image](https://user-images.githubusercontent.com/6437705/82724357-0b1d7300-9d08-11ea-9d0b-acce5a670b0f.png)
![image](https://user-images.githubusercontent.com/6437705/82724352-06f15580-9d08-11ea-9ebf-7e421be17d49.png)

上图是PDF，下图是网页截图。  
生成pdf的时候是否有办法把banner去掉，或者每页截图往上回滚几行

在运行headless chrome的时候执行这段js应该可以把上面的banner去掉  
```javascript
var divs = document.getElementsByTagName('div');
for (var i = 0; i < divs.length; ++i){
    if(divs[i].innerText === "打开APP"){
        divs[i].parentNode.parentNode.style.display="none";
        break;
    }
}
```  

我对go不太懂。直接go get安装也一直没编译过去。报的  
```
# github.com/mmzou/geektime-dl/cli/cmds
go/src/github.com/mmzou/geektime-dl/cli/cmds/cmds.go:50:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/mmzou/geektime-dl/cli/cmds/cmds.go:55:15: cannot use cli.BoolFlag literal (type cli.BoolFlag) as type cli.Flag in slice literal:
        cli.BoolFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/mmzou/geektime-dl/cli/cmds/cmds.go:60:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/mmzou/geektime-dl/cli/cmds/cmds.go:79:8: invalid argument c.Args() (type cli.Args) for len
go/src/github.com/mmzou/geektime-dl/cli/cmds/download.go:31:11: c.Parent undefined (type *cli.Context has no field or method Parent)
go/src/github.com/mmzou/geektime-dl/cli/cmds/login.go:46:19: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/mmzou/geektime-dl/cli/cmds/login.go:51:19: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/mmzou/geektime-dl/cli/cmds/login.go:56:19: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/mmzou/geektime-dl/cli/cmds/login.go:61:19: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/mmzou/geektime-dl/cli/cmds/login.go:66:19: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
        cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/mmzou/geektime-dl/cli/cmds/login.go:66:19: too many errors

```  
这个错误。所以也没法提pr。有懂go的试试看能不能行吗。应该是在`utils/chromedp.go`里面添加执行

--- a/utils/chromedp.go
+++ b/utils/chromedp.go
@@ -57,6 +57,28 @@ func ColumnPrintToPDF(aid int, filename string, cookies map[string]string) error
 				return nil
 			}),
 			chromedp.ActionFunc(func(ctx context.Context) error {
+				s := `
+					var divs = document.getElementsByTagName('div');
+					for (var i = 0; i < divs.length; ++i){
+						if(divs[i].innerText === "打开APP"){
+							divs[i].parentNode.parentNode.style.display="none";
+							break;
+						}
+					}
+				`
+				_, exp, err := runtime.Evaluate(s).Do(ctx)
+				if err != nil {
+					return err
+				}
+
+				if exp != nil {
+					return exp
+				}
+
+				return nil
+			}),
+
+			chromedp.ActionFunc(func(ctx context.Context) error {
 				// time.Sleep(time.Second * 5)
 				var err error
 				buf, _, err = page.PrintToPDF().WithPrintBackground(true).Do(ctx)

